client/landing/jetpack-cloud/sections/scan/main.tsx

```diff
@@ -56,9 +56,12 @@ class ScanPage extends Component< Props > {
 				<SecurityIcon icon="in-progress" />
 				{ this.renderHeader( translate( 'Preparing to scan' ) ) }
 				<p>
-					Lorem ipsum. We need to change this text. The scan was unable to process the themes
-					directory and did not completed successfully. In order to complete the scan you will need
-					to speak to support who can help determine what went wrong.
+					{ translate(
+						'Lorem ipsum. We need to change this text. The scan was unable to process ' +
```

Comment:
Thanks for taking care of the translations Matt. This one, I think we should remove it since it's a placeholder and we don't want to send that for translation.

Reply:
Good point!